diff --git a/dist/index.js b/dist/index.js
index aad810a78758bfb83626707c4609d7201da1b486..b336632d946b17ef287e1eef8313d81c02b08f8b 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -66,13 +66,7 @@ __name(calculateShardId, "calculateShardId");
 
 // src/functions/runtime.ts
 function shouldUseGlobalFetchAndWebSocket() {
-  if (typeof globalThis.process === "undefined") {
-    return "fetch" in globalThis && "WebSocket" in globalThis;
-  }
-  if ("versions" in globalThis.process) {
-    return "deno" in globalThis.process.versions || "bun" in globalThis.process.versions;
-  }
-  return false;
+  return true;
 }
 __name(shouldUseGlobalFetchAndWebSocket, "shouldUseGlobalFetchAndWebSocket");
 
